<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WireGuardProp</class>
 <widget class="QWidget" name="WireGuardProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WireGuard Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Interface</string>
     </property>
     <layout class="QFormLayout" name="form1Layout">
      <item row="0" column="0">
       <widget class="QLabel" name="addressIPv4Label">
        <property name="text">
         <string>Address (IPv4):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="addressIPv4LineEdit">
        <property name="toolTip">
         <string>IPv4 Internet address with
CIDR (example: 10.22.13.123/32)
assigned to the local interface.
IPv4 or IPv6 address (or both) required</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="addressIPv6Label">
        <property name="text">
         <string>Address (IPv6):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="addressIPv6LineEdit">
        <property name="toolTip">
         <string>IPv6 Internet address with
CIDR assigned to the local interface.
(example: fc00:aaaa:aaaa:aa03::1bc9/128) 
IPv4 or IPv6 address (or both) required</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="privateKeyLabel">
        <property name="text">
         <string>Private key:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="PasswordField" name="privateKeyLineEdit">
        <property name="toolTip">
         <string>Required.
A base64 private key generated by wg genkey.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="dNSLabel">
        <property name="text">
         <string>DNS:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="dNSLineEdit">
        <property name="toolTip">
         <string>Optional.
IPv4 or IPv6 address to set the interface's DNS server.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Peer</string>
     </property>
     <layout class="QFormLayout" name="form2Layout">
      <item row="0" column="0">
       <widget class="QLabel" name="publicKeyLabel">
        <property name="text">
         <string>Public key:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="publicKeyLineEdit">
        <property name="toolTip">
         <string>Required.
A base64 public key calculated by wg pubkey 
from a private key, and usually transmitted 
out of band to the author of the configuration file.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="allowedIPsLabel">
        <property name="text">
         <string>Allowed IPs:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="allowedIPsLineEdit">
        <property name="toolTip">
         <string>Required.
A comma-separated list of IP (v4 or v6) addresses 
with CIDR masks from which incoming traffic for 
this peer is allowed and to which outgoing traffic 
for this peer is directed. The catch-all 0.0.0.0/0 
may be specified for matching all IPv4 addresses, 
and ::/0 may be specified for matching all IPv6 addresses.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="endpointLabel">
        <property name="text">
         <string>Endpoint:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="endpointLineEdit">
        <property name="toolTip">
         <string>Optional.
An endpoint IP followed by a colon, and then a port number.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdvanced">
       <property name="text">
        <string>Advanced...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordField</class>
   <extends>QLineEdit</extends>
   <header location="global">passwordfield.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
